src/util/table/DECheckbox.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @author aramsey
+ *
+ * A regular MUI Checkbox but using the theme's primary color
+ */
+import React from "react";
+import { Checkbox } from "@material-ui/core";
+
+function DECheckbox(props) {
+    const { children } = props;
+
+    return (
+        <Checkbox color="primary" {...props}>
```

Comment:
Spreading these props down onto the `Checkbox` will still include the `children` in this spread as well.
It seems this still works fine, but maybe this was what you were intending?
```jsx
function DECheckbox({ children, ...props })
```
This `rest` syntax will separate the `children` out from the rest of the `props`.

Reply:
I get confused with `spread` vs `rest` as well...

Reply:
Oops, yes, that's what I intended.  I should've waited like 5 more minutes before merging 😄

Reply:
Actually, once I saw it was merged, it prompted me to do a quick review 😆